ttps/impact/lotl-ransomware/README.md

```diff
@@ -0,0 +1,51 @@
+# LOTL Ransomware Encryption
+
+![Meta TTP](https://img.shields.io/badge/Meta_TTP-red)
+
+This TTP leverages the `zip` command available on Linux systems to encrypt files in a specified directory, simulating a ransomware attack using tools already present on the machine. The command encrypts the contents of the target directory and requires a password for decryption, illustrating a data encryption impact scenario often used by threat actors.
+
+## Arguments
+
+- **target_dir**: The directory to encrypt.
+
+  Default: /dev/shm
+
+- **encryption_key**: The password used to encrypt the directory.
+
+  Default: password
+
+## Requirements
+
+1. Access to a Linux system where the `zip` and `unzip` commands are available.
+2. Permission to modify files within the target directory.
+
+## Examples
+
+You can run the TTP using the following command (adjust arguments as needed):
+
+```bash
+ttpforge run forgearmory//impact/data-encrypt/LOTL-ransomware/LOTL-ransomware.yaml \
```

Comment:
Looks like the directory to execute should be `forgearmory//impact/ltol-ransomware/lotl-ransomware.yaml`

Reply:
Done